packages/constants/src/constants.ts

```diff
@@ -119,6 +119,7 @@ export const GRAPHQL_TARGETS = {
   MERCHANT_CENTER_BACKEND: 'mc',
   COMMERCETOOLS_PLATFORM: 'ctp',
   DASHBOARD_SERVICE: 'dashboard',
+  CHANGE_HISTORY: 'change-history',
```

Comment:
```suggestion
  CHANGE_HISTORY_SERVICE: 'change-history',
```

For consistency with the dashboard target?

Reply:
+1